publisher/src/components/DataUpload/ShareUploadErrorWarnings.tsx

```
@@ -85,48 +87,32 @@ function ShareUploadErrorWarnings() {
   }, []);
 
   // here will be fetch spreadsheet call and loading and error handling
-  // useEffect(() => {
-  //   const initialize = async () => {
-  //     const result = await reportStore.getSpreadsheetReviewData(spreadsheetId);
-  //     if (result instanceof Error) {
-  //       setLoadingError(result.message);
-  //     }
-  //
-  //     setNewAndUpdatedReports({
-  //       newReports: result.new_reports || [],
-  //       updatedReports: result.updated_reports || [],
-  //       unchangedReports: result.unchanged_reports || [],
-  //     });
-  //
-  //     const errorsWarningsAndMetrics = processUploadResponseBody(
```

Do we need lines 101-109?

---

I dont think so because `errorsWarningsAndMetrics` and `hasErrorsOrWarnings` are already initialized [above](https://github.com/Recidiviz/justice-counts/pull/767/files#diff-007f291ff8b62583d3ffac669b13f55aa82368c45ab3188f4233d031c1b5694dL67) on lines 69 and 70.

---

I think that is using the wrong data though -- we do `import { data } from "../../mocks/spreadsheetReviewData"` at the top of the file, and then on line 67 we use that mock data ` const errorsWarningsAndMetrics = processUploadResponseBody(data);`

---

oh wow thank you so much for catching this

---

Oh nice!!! Completely missed that!